checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -299,9 +299,9 @@ def _render_dynamic_blocks(self) -> None:
                 if dynamic_blocks:
                     try:
                         rendered_blocks = self._process_dynamic_blocks(dynamic_blocks)
-                    except Exception:
+                    except Exception as e:
                         logging.info(f'Failed to process dynamic blocks in file {vertex.path} of resource {vertex.name}'
-                                     f' for blocks: {dynamic_blocks}')
+                                     f' for blocks: {dynamic_blocks}, error: {e}')
```

Comment:
```suggestion
                                     f' for blocks: {dynamic_blocks}', exc_info=True)
```
any reason not to use `exc_info`